### PR TITLE
[CARBONDATA-3760]Enable prefetch in presto to improve query performance

### PR DESCRIPTION
**Pull request by akashrn5**
### Why is this PR needed?
 Prefetch was set to false explicitly.
 
 ### What changes were proposed in this PR?
Improve the query performance with prefetch enabled
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No
TPCDS queries executed and some queries time was improved by almost 50%

### CODE REVIEW

File: integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataPageSource.java [hunk 1]
```diff
@@ -356,7 +356,6 @@ private PrestoCarbonVectorizedRecordReader createReaderForColumnar(HiveSplit car
     QueryModel queryModel = createQueryModel(carbonSplit, tableHandle, columns, conf);
     if (isDirectVectorFill) {
       queryModel.setDirectVectorFill(true);
-      queryModel.setPreFetchData(false);
```
Comment (ajantha-bhat) [inline integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataPageSource.java:359]: @QiangCai : During streaming support for presto [#3001]
you made prefetch as false. 
Any particular reason or issue you found with that?

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2611/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/903/

**Comment by ajantha-bhat** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/908/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2617/

File: integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataPageSource.java (hunk 1 above)
Comment (QiangCai) [reply integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataPageSource.java:359]: No, we can enable it.

Comment (ajantha-bhat) [reply integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataPageSource.java:359]: ok. Thanks

**Comment by ajantha-bhat** [comment]
LGTM

**Pull request closed without merging**